posthog/migrations/0204_global_tags_setup.py

```diff
@@ -0,0 +1,188 @@
+# Generated by Django 3.2.5 on 2022-02-01 22:51
+
+import django.contrib.postgres.fields
+import django.db.models.deletion
+from django.db import migrations, models
+
+import posthog.models.utils
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("posthog", "0203_dashboard_permissions"),
+    ]
+
+    operations = [
+        migrations.RemoveField(model_name="dashboard", name="tags",),
```

Comment:
I think we want these to be `AlterField` migrations. If we `RemoveField` we're going to loose the data.